tutorials/buoyantSimpleFoam-aster/runSolid.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
Could you add a few comments on what is going on here?

Reply:
The `as_run --quick` command is obsolete, and is removed.
I think the other two lines are quite self-explanatory, let me know if you still believe they should have a few comments.

Reply:
Have you already pushed the latest version? I still see `as_run --quick` here. The other two are fine.

Reply:
Yes, this line of code is gone in the `tutorial-addition` branch ([link](https://github.com/precice/code_aster-adapter/blob/tutorial-addition/tutorials/buoyantSimpleFoam-aster/runSolid.sh))